staker/staker.go

```diff
@@ -190,12 +200,21 @@ func (s *Staker) Initialize(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
+	walletAddressOrZero := s.wallet.AddressOrZero()
+	if walletAddressOrZero != (common.Address{}) {
+		balance, err := s.client.BalanceAt(ctx, walletAddressOrZero, nil)
+		if err != nil {
+			return err
+		}
+		stakerBalanceGauge.Update(balance.Int64())
+	}
 
 	if s.blockValidator != nil && s.config.StartFromStaked {
-		latestStaked, _, err := s.validatorUtils.LatestStaked(&s.baseCallOpts, s.rollupAddress, s.wallet.AddressOrZero())
+		latestStaked, _, err := s.validatorUtils.LatestStaked(&s.baseCallOpts, s.rollupAddress, walletAddressOrZero)
 		if err != nil {
 			return err
 		}
+		stakerAmountStakedGauge.Update(int64(latestStaked))
```

Comment:
`latestStaked` is the node number we're staked on, not the amount we're staked with. The latest staked node is actually a great idea for a metric though, we should add this as a separate metric.

Reply:
fixed + added the new metric, thanks